docs_devel/docs/07.ManualContainerTaskBuild.md

```diff
@@ -304,7 +248,11 @@ same location as `docbook-xml-4.5`.
        └────/doc_src_paths.xml
 	   └────/docbook-utf8.xsl
 
+
 # Fonts
+
+-> Aren't the fonts installed in the container?
```

Comment:
They should be. This would apply to building the documentation without using a container, which requires the user to manually install all font and software dependencies.

Reply:
Sure, and if we still have the possibility to make manual build, I don't understand why we have a container requirement.

Reply:
It should probably be called a container _option_.

In theory, a container makes building the documentation easier than the manual process because it… well… ***contains*** everything!

Something strange seems to be going on with the manual build process, though. I'm getting everything inside a literal `${target}` directory in `doc_src\en\` instead of in the correct `docs\en\` folder if I try to build with Ant. More investigation required.

Reply:
> Something strange seems to be going on with the manual build process, though. I'm getting everything inside a literal `${target}` directory in `doc_src\en\` instead of in the correct `docs\en\` folder if I try to build with Ant. More investigation required.

OK. It looks like building the documentation manually now requires passing both the `-Dtarget` and the `-Dlanguage` arguments to Ant.

I'm not sure why it didn't need to be passed explicitly before but is necessary now.

I think it might be a good idea to split this file into separate "using a container to build the documentation" and "building the documentation manually" files.

Reply:
> I think it might be a good idea to split this file into separate "using a container to build the documentation" and "building the documentation manually" files.

I agree. Thank you.

Reply:
I will work it.